src/System Application/App/AI/src/Azure OpenAI/AOAIDeploymentsImpl.Codeunit.al

```diff
@@ -103,12 +108,55 @@ codeunit 7769 "AOAI Deployments Impl"
         exit(GetDeploymentName(GPT4oMiniLatestLbl, CallerModuleInfo));
     end;
 
+    // Initializes dictionary of deprecated models
+    local procedure InitializeDeploymentDeprecationDates()
+    begin
+        if DeprecationDatesInitialized then
+            exit;
+
+        // Add deprecated deployments with their deprecation dates here
+        DeprecatedDeployments.Add(Turbo0301SaasLbl, DMY2Date(1, 11, 2024));
```

Comment:
This label is defined in the file, but it's between the `#if not CLEAN25` and the `#endif` preprocessor directive. That means it will be automatically removed in a couple of major updates. At that point, your code will no longer compile, so you need to add the lines for this model and the other obsoleted models within an `#if not CLEAN25` and an `#endif` as well.

We can talk about this in person if you like :)